README.md

```diff
@@ -58,6 +59,7 @@ The following settings must be passed as environment variables as shown in the e
 | `AWS_REGION` | The region where you created your bucket. Set to `us-east-1` by default. [Full list of regions here.](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions) | `env` | **Yes** | `us-east-1` |
 | `AmplifyAppId` | The unique ID for an Amplify app. For example, `d6a88fjhifqlks` | `secret env` | **Yes** | N/A |
 | `BackendEnvARN` | The Amazon Resource Name (ARN) for a backend environment that is part of an Amplify app. | `secret env` | **Yes** | N/A |
+| `GITHUB_TOKEN` | The GITHUB_TOKEN, required to post the comment with the preview URL | `github env` | **Yes** | N/A |
```

Comment:
This field is not required, right? Please change ```**Yes**``` to No

Reply:
I was thinking of keeping it as a required field, what do you think? That way, the comments will come as default.

Either we keep it as optional, or I can update `entrypoint.sh` accordingly. What do you think?

Reply:
I think keep it as an optional field would be better. Let others to decide if they want to show the preview link to public by themselves, instead of showing it mandatory.

Reply:
I've made the change.